src/improvements/single.just

```diff
@@ -15,16 +15,33 @@ export SCRIPT_NAME := shell(cmd)
 export signatures := env_var_or_default('SIGNATURES', '')
 export forkBlockNumber := env_var_or_default('FORK_BLOCK_NUMBER', '-1')
 
-simulate hdPath='0':
+# 'senderAddress' is the address you want to run this simulation as.
+# The 'senderAddress' will be the 'from' field in the Tenderly simulation. 
+simulate senderAddress="" hdPath='0':
   #!/usr/bin/env bash
 
   config=${TASK_PATH}/config.toml
   script=${SCRIPT_PATH}/template/${SCRIPT_NAME}.sol
 
   echo "RPC URL: ${rpcUrl}"
-
   echo "Using script ${script}"
-  echo ""
+
+  echo "Setting sender address..."
+  # Default fallback sender: address(uint160(uint256(keccak256("foundry default caller"))))
+  default_sender=0x1804c8AB1F12E6bbf3894d4083f33e07309d1f38
```

Comment:
shouldn't the default_sender get an owner [from the safe owners like here](https://github.com/ethereum-optimism/superchain-ops/blob/15397a1c4034e008e5fc89c9e5136bcffc250944/src/improvements/nested.just#L31)?

Reply:
If we back up for a second, what is the reason we want to see the 'from' address in Tenderly be an owner of a safe or a ledger? 
I agree, it looks nice but I don't think it's adding anything? Ultimately the state overrides mean the sender can be from the default foundry account. 

If I add the line in you're talking about we're effectively removing a feature that we wanted in the new superchain-ops i.e. no having to enter the safe addresses in .env or config files.